data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -56,14 +57,27 @@ def get_bq_json_for_survey_questions_response_json(
     }
 
 
-def get_survey_answer_details(access_token: str, survey_id: str) -> list:
+def get_survey_answers_in_next_page(
+    access_token: str,
+    page_url: str
+) -> dict:
+    headers = get_surveymonkey_api_headers(access_token)
+    return requests.get(page_url, headers=headers).json()
+
+
+def iter_survey_answer_detail_response(access_token: str, survey_id: str) -> Iterable[dict]:
     headers = get_surveymonkey_api_headers(access_token)
     response = requests.get(
-        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/responses/bulk',
+        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/responses/bulk/?per_page=100',
```

Comment:
Just noticing that SurveyMonkey seem to be the using the term `responses` or `response` in the URL (instead of `answers`).

Reply:
yes but to avoid the confusion between api response and survey response, decided to use answers for survey responses as there is no difference in meanings.